src/applications/hca/reducers/hca-enrollment-status-reducer.js

```diff
@@ -62,9 +62,10 @@ function hcaEnrollmentStatus(state = initialState, action) {
       const { errors } = action;
       const noESRRecordFound =
         errors && errors.some(error => error.code === '404');
-      const hasServerError = errors && errors.some(error => error.code >= 500);
       const hasRateLimitError =
         errors && errors.some(error => error.code === '429');
+      // if the error is not given special handling, treat it like a server error
+      const hasServerError = !noESRRecordFound && !hasRateLimitError;
```

Comment:
For the sake of clarity it could be worth while finding a new name for this variable, since it no longer explicitly checks for 5xx codes.

Reply:
True. But I'm going to hold off for now since that would require adjusting a few more files owned by other teams :-/